2019/day1/day1.go

```diff
@@ -0,0 +1,62 @@
+package day1
+
+import (
+	"strconv"
+)
+
+func Part1(moduleMasses []string) (int, error) {
```

Comment:
In these examples, you are better off pre-converting the data. It means you won't need to propagate string conversion errors.

Reply:
much better.